api/maketdata.go

```diff
@@ -144,10 +148,14 @@ func makeChartsRoute(router gin.IRouter) {
 			ginutils.RenderError(c, http.StatusInternalServerError, "Invalid time_start")
 			return
 		}
+		maxItems, err := strconv.Atoi(c.Query("max_items"))
+		if err != nil || maxItems <= 0 {
+			maxItems = defaultMaxChartItems
+		}
 
 		currency := c.DefaultQuery("currency", blockatlas.DefaultCurrency)
 
-		chart, err := charts.GetChartData(uint(coinId), token, currency, timeStart)
```

Comment:
Add documentation to this parameter

Reply:
run `swag init` after adding this parameter in the documentation